packages/wallet/src/wallet.ts

```diff
@@ -1,28 +1,35 @@
+import { Address, getAddress, isAddress } from "viem";
 import { AdvanceRequestHandler, Voucher } from "@deroll/app";
-import {
-    Address,
-    encodeFunctionData,
-    erc20Abi,
-    getAddress,
-    isAddress,
-} from "viem";
-
-import { cartesiDAppAbi, dAppAddressRelayAddress } from "./rollups";
-import {
-    isERC20Deposit,
-    isEtherDeposit,
-    parseERC20Deposit,
-    parseEtherDeposit,
-} from ".";
+
+import { ether, erc20, erc721, erc1155Single, erc1155Batch } from "./contracts";
+
+import { depositHandler } from "./token";
+import { RelayError } from "./errors";
 
 export type Wallet = {
     ether: bigint;
-    erc20: Record<Address, bigint>;
+    erc20: Record<string, bigint>;
```

Comment:
is the string in this case required to be an Address?
If yes, it should be an Address type.

Reply:
Some address can be alias, but in some cases, like withdraw to generate voucher, need to be a Address type.
So, when you need this, transfer for a real address, and call withdraw method.
This alias follow this pattern: https://github.com/tuler/deroll/blob/903d8ec54ef7ba39a127b8cb4e61879a495be4f8/packages/wallet/src/wallet.ts#L58